backend/src/Service/Admin/Order/AdminOrderService.php

```diff
@@ -1954,4 +1960,127 @@ public function getCaptainsWhoDeliveredLastFiveOrders(): array
+    }
+
+    /**
+     * Create order/s by admin for testing/debugging purposes
+     */
+    public function createTestingOrderByAdmin(TestingOrderCreateByAdminRequest $request, int $userId): CanCreateOrderResponse|array|string
+    {
+        $response = [];
+
+        if ((! $request->getOrdersCount())
+            || ($request->getOrdersCount() <= 0)
+            || ($request->getOrdersCount() > 3)) {
+            $request->setOrdersCount(1);
+        }
+
+        // 1 check if store owner profile does exist
+        $storeOwnerProfile = $this->storeOwnerProfileService->getStoreOwnerProfileEntityByIdForAdmin($request->getStoreOwner());
+
+        if ($storeOwnerProfile === null) {
+            return StoreProfileConstant::STORE_OWNER_PROFILE_NOT_EXISTS;
+        }
+
+        // 2 check if store branch exist
+        $branch = $this->storeOwnerBranchService->getBranchEntityById($request->getBranch());
+
+        if (! $branch) {
+            return StoreOwnerBranch::BRANCH_NOT_FOUND;
+        }
+
+        $request->setStoreOwner($storeOwnerProfile);
+        $request->setBranch($branch);
+
+        for ($counter = 0; $counter < $request->getOrdersCount(); $counter++) {
+            // 3 check if store has current active subscription
+            $canCreateOrder = $this->subscriptionService->canCreateOrder($storeOwnerProfile->getStoreOwnerId());
+
+            if ($canCreateOrder === StoreProfileConstant::STORE_OWNER_PROFILE_INACTIVE_STATUS
+                || $canCreateOrder->canCreateOrder === SubscriptionConstant::CAN_NOT_CREATE_ORDER) {
+                return $canCreateOrder;
+            }
+
+            $orderCreateByAdminRequest = $this->autoMapping->map(TestingOrderCreateByAdminRequest::class,
+                OrderCreateByAdminRequest::class, $request);
+
+            // 4 check if store subscription remaining cars finished
+            $orderCreateByAdminRequest->setIsHide(OrderIsHideConstant::ORDER_SHOW);
+
+            if ($canCreateOrder->subscriptionStatus === SubscriptionConstant::CARS_FINISHED) {
+                $orderCreateByAdminRequest->setIsHide(OrderIsHideConstant::ORDER_HIDE_TEMPORARILY);
+            }
+
+            // 5 set the delivery date equals to current datetime + 3 minutes just for affording the late in persisting
+            // the order to the database if it is happened
+            $orderCreateByAdminRequest->setDeliveryDate((new DateTime('+ 3 minutes'))->format('Y-m-d H:i:s'));
+
+            $orderCreateByAdminRequest->setPayment(OrderTypeConstant::ORDER_PAYMENT_CASH);
+            $orderCreateByAdminRequest->setOrderCost(OrderCostDefaultValueConstant::ORDER_VALUE_CONST);
+            $orderCreateByAdminRequest->setNote(OrderNoteDefaultValueConstant::ORDER_NOTE_DEFAULT_VALUE_CONST);
+            $orderCreateByAdminRequest->setDestination(OrderClientLocationConstant::ORDER_CLIENT_LOCATION_ARRAY_CONST[array_rand(OrderClientLocationConstant::ORDER_CLIENT_LOCATION_ARRAY_CONST)]);
```

Comment:
Please move this 'random' to a separate function, as same as what you will do for the other random values.
This is for future upgrades to this sweet function.